src/devices/Ad7193/category.txt

```diff
@@ -0,0 +1,3 @@
+adc
+spi
+sigma-delta
```

Comment:
sigma-delta? please remove spi, spi is currently for things exposing SPI, i.e.: SoftwareSpi or FT4222

Reply:
Sigma-delta is a one of the two common types of ADCs.